### PR TITLE
 Closes #221. Ticket creation via voice api.

### LINKED ISSUE #221: Zendesk Voice API support
There seems to be no access to the Zendesk Voice API. I am looking for a way to create voice tickets. Can it be done with this client? If yes, how? And if not, what would be the best practice with PHP.

### PR DESCRIPTION
**Pull request by mindreader**
Wrote a small unit test, ran phpcs on tests and source, ran in own code, and it seems to work.  Based off of PhoneNumber.

### CODE REVIEW

File: src/Zendesk/API/Resources/Voice/Tickets.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUpRoutes()
+    {
+        $this->setRoute('create', "{$this->getResourceName()}.json");
+    }
+
+    public function create(array $params, $routeKey = __FUNCTION__)
```
Comment (joseconsador) [inline src/Zendesk/API/Resources/Voice/Tickets.php]: Docbloc here and this should be 👍

File: src/Zendesk/API/Resources/Voice/Tickets.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+use Zendesk\API\Traits\Resource\Create;
+
+/**
+ * Class Tickets
+ * https://developer.zendesk.com/rest_api/docs/voice-api/voice_integration
+ */
+class Tickets extends ResourceAbstract
+{
+    use Create;
```
Comment (samgavinio) [inline src/Zendesk/API/Resources/Voice/Tickets.php]: This trait doesn't seem to have use since it's immediately overridden in this resource.

**Comment by mindreader** [comment]
Changes as requested.  Sorry for the delay.

**Pull request closed without merging**